deployment/terraform/strings.go

```diff
@@ -70,38 +70,64 @@ pid /run/nginx.pid;
 include /etc/nginx/modules-enabled/*.conf;
 
 events {
-	worker_connections 16384;
-	use epoll;
+  worker_connections 16384;
```

Comment:
We have been going back and forth on spaces vs tabs. Let's commit to one :P

Reply:
Oh, were we? I am space guy :rocket:

Reply:
Oh I see I was misremembering. I was referring to this PR: https://github.com/mattermost/mattermost-load-test-ng/commit/bd7a55e5229bade61d9a998c7b5850927e02dbc9. It seems like you did for some lines, but left others untouched. No worries. :+1:

Reply:
I think that was merely copying and pasting stuff without really bothering. Now it looks much better, everything is aligned properly.